algrtm/src/components/ContactDescription.js

```diff
@@ -4,15 +4,16 @@ import CustomLink from "./shared/CustomLink";
 const ContactDescription = () => {
   return (
     <section className="flex flex-col gap-6 max-w-[22.375rem] w-full">
-      <h2 className=" h2 text-neutralLight-neutral10 leading-[4rem] ">
+      <h2 className=" h4 md:h2SB xl:h2 text-neutralLight-neutral10 leading-[4rem] ">
         Contact
       </h2>
       <div className="flex flex-col gap-4">
-        <h6 className="h6 text-neutralLight-neutral20 leading-[1.98rem]">
+        <h6 className="h6_400 xl:h6 text-neutralLight-neutral20 leading-[1.98rem]">
           Let's explore how we can work together. Share your algorithmic ideas,
           and we’ll provide the expertise to bring them to life.
         </h6>
-        <div className="flex gap-4 items-start">
+
+        <div className=" flex flex-col gap-4 self-stretch xl:flex-row">
```

Comment:
Links `Book a meeting` and `Go to the questionnaire` should not be centered on medium devices:

![image](https://github.com/Nospoko/algrtm/assets/26410548/b932db48-2661-4f13-8ec3-5c7fab4c5470)